tvbwidgets/core/hpc/launcher.py

```diff
@@ -190,9 +191,24 @@ def submit_job(self, executable, inputs, do_stage_out):
         else:
             LOGGER.info('You can use "PyUnicore Tasks Stream" tool to monitor it.')
 
+    @staticmethod
+    def read_file_from_hpc(job, file_name):
+        content = job.working_dir.listdir()
+        storage_config_file = content.get(file_name)
+        if storage_config_file is None:
+            LOGGER.error(f"Could not find file: {file_name}")
+            return None
+        return storage_config_file.raw().read()
+
     def monitor_job(self, job):
         LOGGER.info('Waiting for job to finish...'
                     'It can also be monitored interactively with the "PyUnicore Tasks Stream" tool.')
+        while job.properties['status'] != Status.SUCCESSFUL:
+            while job.properties['status'] != 'RUNNING':
```

Comment:
what happens if then job end in error (instead of success)? 
Isn't here and infinite loop ?